src/components/TransferForm/index.tsx

```diff
@@ -133,10 +192,64 @@ export const TransferForm: React.SFC<TransferFormProps> = ({
               </label>
             </div>
             <div className="col-sm-8">
-              <WalletSelect
-                options={walletStore.getWalletsExceptOne(transfer.from)}
+              <Select
+                optGroups={[
+                  {
+                    label: 'Trading Wallets',
+                    options: walletStore
```

Comment:
i know this is not the scope of the task, but i believe it will be better to rewrite it this way `walletStore.trading(trusted)Wallets.getWalletsExceptOne()`. It will evolve the idea of introducing computed trading/trustedWallets prop in the store, mentioned in prev comment